src/client/core/evaluator.ts

```diff
@@ -0,0 +1,50 @@
+import { typed, create, all, factory } from 'mathjs'
+import * as checker from './../../types/checker'
+
+const config = {}
+const factories = {
+  ...all,
+  // Override for string comparison
+  createEqual: factory(
+    'equal',
+    [],
+    () => (a: string | number, b: string | number) => {
+      return a === b
+    }
+  ),
+  createUnequal: factory(
+    'unequal',
+    [],
+    () =>
+      function unequal(a: string | number, b: string | number) {
+        return a !== b
+      }
```

Comment:
nitpick - arrow function for consistency with lines 11-13?
```suggestion
    () => (a: string | number, b: string | number) => {
      return a !== b
    }
```